src/main/java/org/lwes/emitter/MulticastEventEmitter.java

```diff
@@ -44,27 +44,15 @@
  * @author Anthony Molinaro
```

Comment:
now that you are changing this, can you add a method to control loopback too? http://download.java.net/jdk7/archive/b123/docs/api/java/net/MulticastSocket.html#setLoopbackMode(boolean)